src/modules.json

```diff
@@ -56,7 +56,11 @@
     },
     "assert": {
       "js_file": "js/assert.js",
-      "require": ["util"]
+      "require": ["util", "comparisons"]
+    },
+    "comparisons": {
```

Comment:
An internal module doesn't been expected to be in public scope. An internal prefix might help.

Reply:
Yes, I change it to internal_comparisons. Thanks! :)